src/main/java/com/ticketservice/annotation/NullableWarningAnnotation.java

```diff
@@ -0,0 +1,15 @@
+package com.ticketservice.annotation;
+
+import java.lang.reflect.Field;
+
+public class NullableWarningAnnotation {
+
+    public static void checkFieldsForNull(Object object) {
+        Field[] fields = object.getClass().getDeclaredFields();
+        for (int i = 0; i < fields.length; i++) {
+            if (fields[i] == null) {
```

Comment:
This logic is not hundred percent correct because we are interested in the value of field and not field itself:
So, you should get filed value, compare it with null, and if it's equals null, then print a message